docs/commands/bundle.md

```diff
@@ -1,117 +1,163 @@
-If the file specified as the bundler's output already exists, it will be overwritten.
-</div>
+### Bundle multiple API definitions
 
-### How to bundle a single API definition
-
-This command produces a bundled file in JSON format at the path `dist/openapi.json` starting from the root API definition file `openapi/openapi.yaml`.
+This command creates one bundled file for each of the specified entrypoints in the `dist/` folder. Bundled files are in JSON format.
 
+```bash request
+openapi bundle --output dist --ext json openapi/openapi.yaml openapi/petstore.yaml
 ```
-openapi bundle --output dist/openapi.json openapi/openapi.yaml
+
+```bash output
+dist/openapi.json
+dist/petstore.json
 ```
 
-### How to bundle multiple API definitions
+### Create a fully dereferenced bundle
 
-This command creates one bundled file for each of the specified entrypoints in the folder `dist/`. Bundled files are in JSON format.
+:::warning Note
+JSON output only works when there are no circular references.
+:::
 
-```
-openapi bundle --output dist --ext json openapi/openapi.yaml openapi/petstore.yaml
+```bash
+openapi bundle --dereferenced --output dist --ext json openapi/openapi.yaml openapi/petstore.yaml
 ```
 
-The output will be:
+### Custom configuration file
 
-```
-dist/openapi.json
-dist/petstore.json
+By default, the CLI tool looks for a `.redocly.yaml` configuration file in the current working directory. Use the optional `--config` argument to provide an alternative path to a configuration file. 
+
+```bash
+openapi bundle --config=./another/directory/config.yaml
 ```
 
-### How to make a fully dereferenced bundle
+### Format
 
-<div class="warning">
-JSON output only works when there are no circular references.
-</div>
+#### Codeframe (default)
 
+```bash request
+openapi bundle pet.yaml store.yaml -o ./bundled --format=codeframe
+## equivalent to: openapi bundle pet.yaml store.yaml -o ./bundled
 ```
-openapi bundle --dereferenced --output dist --ext json openapi/openapi.yaml openapi/petstore.yaml
+
+```bash output
+...
 ```
 
-### Options
+Please note that the problems are displayed in the following format: `file:line:column`. For example, `petstore-with-errors.yaml:16:3`.
 
-#### Skip preprocessor
+Depending on the terminal emulator you use, it may be possible to directly click this indicator to edit the file in place.
 
-You may want to skip specific preprocessors upon running the command.
+#### Stylish
 
-```
-openapi bundle --skip-preprocessor=discriminator-mapping-to-one-of,another-example
+```bash request
+openapi bundle pet.yaml store.yaml -o ./bundled --format=stylish
 ```
 
-Learn more about [preprocessors](../resources/custom-rules.md).
+```bash output
+...
+```
 
-#### Skip rule
+In this format, `bundle` shows the file name, line number, and column where the problem occurred. However, the output is compressed and omits other contexts and suggestions.
 
-You may want to skip specific rules upon running the command.
+#### Json
```

Comment:
question: I wonder if it should be JSON (all caps)?

Reply:
Absolutely. Thanks